VK_App/VK_App/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,65 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// NetworkService - класс для работы с сетью
+final class NetworkService {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let baseUrl = "https://api.vk.com"
+        static let path = "/method"
+        static let userID = "user_id"
+        static let accessToken = "access_token"
+        static let extended = "extended"
+        static let ownerId = "owner_id"
+        static let vKey = "v"
+        static let qKey = "q"
+    }
+
+    enum Methods: String {
+        case groups = "/groups.get"
+        case searchGroups = "/groups.search"
+        case friendsList = "/friends.get"
+        case photo = "/photos.getAll"
+    }
+
+    // MARK: - Public Properties
+
+    static let instance = NetworkService()
+
+    var token: String?
+    var userId: String?
+
+    // MARK: - Private Initializers
+
+    private init() {}
+
+    // MARK: - Public Method
+
+    func getResponse(method: Methods) {
+        guard
+            let token = token,
+            let userId = userId
+        else {
+            return
+        }
+
+        let parameters: Parameters = [
+            Constants.userID: userId,
+            Constants.accessToken: token,
+            Constants.extended: "1",
+            Constants.ownerId: "-25380626",
+            Constants.vKey: "5.131",
+            Constants.qKey: "i"
```

Comment:
```suggestion
            Constants.extended: "",
            Constants.ownerId: "",
            Constants.vKey: "",
            Constants.qKey: ""
```
текстовки в константы